mindmap/mindmap.py

```diff
@@ -180,65 +226,52 @@ def connect_to_s3() -> Tuple[boto3.client, str]:
 
         return s3_client, aws_bucket_name
 
-    def get_file_key(self, anonymous_user_id: str) -> str:
+    def get_file_key(self, suffix: str) -> str:
         """
         Return the key (path) to save and retrieve the file in S3.
 
-        Returns:
-            str: The key (path) to use in S3.
-        """
-        block_id = self.location.block_id # pylint: disable=no-member
-        return f"mindmaps/{block_id}/{anonymous_user_id}/mindmap.json"
-
-    def file_exists_in_s3(self, anonymous_user_id: str) -> bool:
-        """
-        Check if the file exists in S3.
+        Args:
+            suffix (str): The suffix to use in the key (path).
```

Comment:
can we change the naming? path_prefix? or path_suffix? something that gives more away from what it is